src/api/config.js

```diff
@@ -4,12 +4,12 @@ import { getAuth } from 'firebase/auth';
 
 // Your web app's Firebase configuration
 const firebaseConfig = {
-	apiKey: 'FILL_ME_IN',
-	authDomain: 'FILL_ME_IN',
-	projectId: 'FILL_ME_IN',
-	storageBucket: 'FILL_ME_IN',
-	messagingSenderId: 'FILL_ME_IN',
-	appId: 'FILL_ME_IN',
+	apiKey: 'AIzaSyC7Va6r7PL9JZrTz8jn1Lwqwr_yhSAs2do',
```

Comment:
Are we supposed to have an api key in this file?

Reply:
We've generally left it in this file since the key usually expired after a certain period anyway (90 days I think)